test/suite_implementation.js

```diff
@@ -12,6 +17,7 @@ module.exports = function(style, options, _callback) {
         }
     }
 
+    window.XMLHttpRequest = XMLHttpRequest;
```

Comment:
The built-in jsdom XHR doesn't work but I don't fully understand why. I'll dig in now.